### DISCUSSION TITLE
Working of Discussion answered badge under Highlight Badges

### DISCUSSION THREAD

**Discussion by Anmol-Baranwal** [drknzz/GitHub-Achievements#16] [Q&A] [answered]
There is a section of [highlight badges](https://github.com/drknzz/GitHub-Achievements#-highlights-badges-), under which there is a badge of   ![discussion-answered-dark](https://user-images.githubusercontent.com/65187002/173078083-15a75f15-b040-4a92-8d70-561a206d9fd9.svg#gh-dark-mode-only) for `Discussion answered`.

You can see my profile [here](https://github.com/Anmol-Baranwal) that I have answered two discussions but I didn't get the badge.

So, can someone please explain how this specific badge works?

 I am not asking solely for the badge but I am rather curious since there is also official documentation [here](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#displaying-badges-on-your-profile)

**Comment by drknzz** [comment, marked as answer]
Hey, thanks for asking.
From my little investigation it seems that they've removed it?
I've browsed a few of really active profiles that should have the check mark for discussions on their profile yet there is none.

**Reply by drknzz** [reply]
It might be because there is now an acutal achievement for answering discussions? Or I am wrong and they are still earnable? You could post link to a profile with it if you manage to find one.

**Reply by Anmol-Baranwal** [reply]
They should update the official documentation. I asked it in the official discussions of GitHub, someone said that:

`The little checkmark for discussions answered that used to show up under highlights had been replaced by the Galaxy Brain Achievement Badge`.

Visit the discussion at the following link: 
```
https://github.com/community/community/discussions/34876
```

**Comment by 1RB** [comment]
The little checkmark for answered discussions that used to show up under highlights has been replaced by the Galaxy Brain Achievement Badge

*if this helped please mark it as the answer as I'm trying to get galaxy brain achievement myself*

**Comment by dev-aniketj** [comment]
I applied for the github student pack and it is approved
I don’t have a pro badge on my github profile
https://github.com/dev-aniketj

**Reply by Anmol-Baranwal** [reply]
Are you sure it has been approved?

If there is still a problem, I recommend going [here](https://github.com/community/community/discussions) to ask or see if anyone has faced similar problem
